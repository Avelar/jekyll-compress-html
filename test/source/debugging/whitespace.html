---
layout: compress
---

<p>One Two  Three   </p>
