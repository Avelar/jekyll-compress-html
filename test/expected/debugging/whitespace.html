<!-- compress_html.debugging: size:25, endsize:17 --><p>One Two Three
